webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/PowerStatus/PowerStatusDropDown.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Tooltip,
+} from '@patternfly/react-core';
+
+import { foremanUrl } from '../../../../common/helpers';
+import { useAPI } from '../../../../common/hooks/API/APIHooks';
+import { translate as __ } from '../../../../common/I18n';
+import PowerStatusIcon from './PowerStatusIcon';
+import {
+  POWER_REQUEST_OPTIONS,
+  BASE_POWER_STATES,
+  SUPPORTED_POWER_STATES,
+} from './constants';
+import { changeHostPower } from './actions';
+import { openConfirmModal } from '../../../ConfirmModal';
+
+import '../styles.scss';
```

Comment:
Also, 
Won't it be loaded in webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js?

Reply:
Yes, though I didn't remove this import for keeping the power status component reuseable.